README.md

```diff
@@ -1,13 +1,46 @@
+
+### Globalization
+The React TimePicker component supports globalization like internationalization and localization that updates the time format value in the pop-up list to match a specified culture.
+
+### Time interval
+Increment or decrement each time value of the selection list by one step at a time. You can change the steps based on your needs.
+
+## Related links
+[Learn More about React TimePicker](https://www.syncfusion.com/react-components/react-timepicker?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples)
+[Download Free Trial](https://www.syncfusion.com/downloads/react?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples)
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples)
+[Documentation](https://ej2.syncfusion.com/react/documentation/timepicker/getting-started?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples)
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-the-react-timepicker?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples)
+[Community Forums](https://www.syncfusion.com/forums/react-js2?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples)
+[Suggest a feature](https://www.syncfusion.com/feedback/react?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples)
+[Online Examples](https://ej2.syncfusion.com/react/demos/#/bootstrap5/timepicker/default?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples)
+
+## About Syncfusion React Components
+
+Syncfusion's [React Components](https://www.syncfusion.com/react-components?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples) library is the only suite that you will ever need to build an application since it contains over 80 high-performance, lightweight, modular, and responsive UI Components in a single package. In addition to TimePicker, we provide popular React Components such as [DataGrid](https://www.syncfusion.com/react-components/react-grid?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples), [Charts](https://www.syncfusion.com/react-components/react-charts?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples), [Scheduler](https://www.syncfusion.com/react-components/react-scheduler?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples), [Diagram](https://www.syncfusion.com/react-components/react-diagram?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples), and [Word Processor](https://www.syncfusion.com/react-components/react-word-processor?utm_source=github&utm_medium=listing&utm_campaign=react-timepicker-github-samples)
+
```

Comment:
Use this "Time Picker"
![image](https://github.com/SyncfusionExamples/getting-started-with-the-react-timepicker/assets/95272441/71c0ace6-cfca-46b6-b037-ca08939c11f8)